### ISSUE TITLE
Colorscheme inverted for folder path

### ISSUE THREAD

**Issue by dylanpjx** [vifm/vifm#793]
I'm trying to use the OneDark colorscheme. I'm not sure what's wrong but the colorscheme is always inverted for the folder path and the statusline. 

![image](https://user-images.githubusercontent.com/42106466/167527793-73c8ac4e-cd52-4812-b6e5-33dd4ac2696e.png)

This is what happens when I change colorscheme to Default
![image](https://user-images.githubusercontent.com/42106466/167527992-dcc424c5-b670-44f8-a7f7-2ab6e7b45ecd.png)

Term: xterm-256 (Alacritty)
tput colors: 256

Could someone shed some light on this issue?

**Comment by xaizek** [comment]
It's a bug in v0.10.1

See https://github.com/vifm/vifm/issues/557#issuecomment-642801973 for a workaround or upgrade to a newer version if possible.

**Comment by dylanpjx** [comment]
I have tried both the solutions and it still looks like the above.
![image](https://user-images.githubusercontent.com/42106466/167595215-4ca8480f-d46b-4073-82d8-95830d38d089.png)

**Comment by xaizek** [comment]
Hm, please post output of `:highlight OtherWin` when the issue is visible.

**Comment by dylanpjx** [comment]
`OtherWin   cterm=none ctermfg=default ctermbg=default`

**Comment by xaizek** [comment]
`onedark` is taken from https://github.com/vifm/vifm-colors/blob/master/onedark.vifm, right? Please attach `$VIFM/colors/Default.vifm` and possibly `onedark.vifm` if it differs from the link above. Are you switching colorschemes with `:colorscheme name` command? Interactively or in `vifmrc`? I'm trying to reproduce this on v0.12, but it doesn't show up so far.

**Comment by dylanpjx** [comment]
Yes, all my colorschemes are taken from the above link and placed into `~/.config/vifm/colors`. 

Default.vifm: (I removed all occurrences of `cterm=bold,underline,reverse,standout,italic`, I have also tried undoing the changes but to no avail) 
![image](https://user-images.githubusercontent.com/42106466/167602292-f3c8eed1-ec92-40da-97eb-21ea9fcb316c.png)

I have the colorscheme to be set to OneDark on vifmrc. For testing, I tried switching back and forth interactively using `:colorscheme` but both resulted in the inverted colors.

Could this be related to Alacritty's colorscheme somehow messing with this?

**Comment by xaizek** [comment]
> Could this be related to Alacritty's colorscheme somehow messing with this?

Unlikely, it would show up in both panes, you can try `xterm` or whatever other terminal is installed on your system. I suspect it will look the same, it looks like a bug.

What happens if after starting Vifm you do `:highlight clear`?

**Comment by dylanpjx** [comment]
I did `:highlight clear` after starting and it works as shown:
![image](https://user-images.githubusercontent.com/42106466/167753202-d19efcd1-1f1b-4753-95b7-8f2e3dd24f65.png)

**Comment by xaizek** [comment]
Have you tried other terminals just in case? I tried installing `alacritty` using `nix`, but it won't start.

You can try running `:highlight OtherWin ctermfg=default ctermbg=default` after `:highlight clear` to see if that causes the issue to appear and whether changing second command to `:highlight OtherWin cterm=none ctermfg=default ctermbg=default` makes any difference.

**Comment by dylanpjx** [comment]
I have tried it with `gnome-term` and the result is the same. Both commands don't change anything after running `:highlight clear`

**Comment by xaizek** [comment]
Thanks. Those are basically first and last commands of `Default.vifm`. Please try commenting out the rest of the file and see if the issue persists. If it doesn't appear, try uncommenting other lines: `Win` can potentially affect the result, the rest shouldn't, but who knows.

Another thing you might try is running `vifm --no-configs` (quit with `:q!`) and then doing `:colorscheme onedark` and `:colorscheme Default`. Maybe some of your settings in `vifmrc` affect the issue, which is why I can't reproduce it.

**Comment by dylanpjx** [comment]
The default colorscheme seems to be ok now, but `:colorscheme onedark` is still buggy. I doubt this would be the reason but I'm using [vifmrun](https://github.com/cirala/vifmimg/blob/master/vifmrun) to launch vifm.

**Comment by xaizek** [comment]
Using your configuration doesn't trigger it either... That `OneDark.vifm` doesn't have `OtherWin`, maybe try adding
```vim
highlight OtherWin ctermfg=default ctermbg=default
```
or
```vim
highlight OtherWin cterm=none ctermfg=default ctermbg=default
```
It shouldn't make any difference, but who knows where the bug is.

**Comment by dylanpjx** [comment]
Nope, the above also doesn't seem to work. Haha thanks for all the help though!
